addon/components/phone-input.js

```diff
@@ -44,6 +46,17 @@ export default Component.extend({
     */
     this.number = this.number || null
 
+    /**
+      Whether or not to allow the dropdown. If disabled, there is no dropdown arrow, and the selected flag is not clickable. Also we display the selected flag on the right instead because it is just a marker of state.
+
+      @argument allowDropdown
+      @type {boolean}
+    */
+
+    this.allowDropdown = isPresent(this.allowDropdown)
```

Comment:
Could we make it an opt-in instead? :pray:

If we make it an opt-in, we won't need a major release (it would be breaking). Also, we would remain inline with [intl-tel-input](https://github.com/jackocnr/intl-tel-input#options) where the default is true.